### PR TITLE
use community release parent

### PR DESCRIPTION
**Pull request by akeller**
Since I'm still struggling to get this one to appear in Sonatype Nexus, this was my next step in debugging - Use the Community release parent and adjust the profile to match the other project you were having release issues with 🤷‍♀️ 

Can you tell I'm not a Java/maven dev? 😆

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -196,34 +196,35 @@
...
-      <properties>
-        <plugin.version.gpg>1.6</plugin.version.gpg>
-        <gpg.passphrase>${env.GPG_PASS}</gpg.passphrase>
-        <resume>false</resume>
-        <tag>${env.RELEASE_VERSION}</tag>
-        <releaseVersion>${env.RELEASE_VERSION}</releaseVersion>
-        <developmentVersion>${env.DEVELOPMENT_VERSION}</developmentVersion>
-        <arguments>-DskipTests</arguments>
-      </properties>
+      <id>community-action-maven-release</id>
```
Comment (pihme) [inline pom.xml:201]: :x: FYI this name is referenced in the GitHub action https://github.com/camunda-community-hub/zeebe-process-generator/blob/3b758dbeb2f81d39e24629b4aa558dcca1418395/.github/workflows/ci.yml#L54

I will add a commit for this.

File: pom.xml [hunk 2]
```diff
@@ -196,34 +196,35 @@
...
-              <configuration>
-                <gpgArguments>
-                  <arg>--pinentry-mode</arg>
-                  <arg>loopback</arg>
-                </gpgArguments>
-              </configuration>
-            </plugin>
-          </plugins>
-        </pluginManagement>
+        <plugins>
```
Comment (pihme) [inline pom.xml:203]: :thinking: I wonder whether we need this at all. This profile is defined in https://github.com/camunda-community-hub/community-hub-release-parent/blob/0f52943897c35d8012966c9b087a507d79b33697/pom.xml#L102-L120

Here we overwrite some stuff, but nothing really important. 

https://github.com/camunda-community-hub/zeebe-process-generator/issues/95

**Comment by pihme** [comment]
> Can you tell I'm not a Java/maven dev?

Not at all, you are a natural :guitar:

File: pom.xml (hunk 1 above)
Comment (pihme) [reply pom.xml:201]: Commit added

**Review by pihme — APPROVED** [review]

**Comment by pihme** [comment]
@akeller 

With this change I was able to get something to Nexus:
image

Not perfect though: The release is marked as open, and you can only see what has been released if you open the content. 

Nevertheless, from there I could manually close the release, then wait some time, then release it. (I assume it will find its way into Maven Central)

File: pom.xml (hunk 2 above)
Comment (akeller) [reply pom.xml:203]: This is where I really wasn't sure.

**Pull request merged**